docs/releasenotes/10.4.0.rst

```diff
@@ -42,6 +42,14 @@ The ``hints`` parameter in :py:meth:`~PIL.ImageDraw.getdraw()` has been deprecat
 API Changes
 ===========
 
+ImageFont.truetype
+^^^^^^^^^^^^^^^^^^
+
+:py:func:`~PIL.ImageFont.truetype` now searches for fonts in the directory indicated
+by the ``XDG_DATA_HOME`` environment variable in addition to those specified in
+``XDG_DATA_DIRS``.  Additionally, the default for ``XDG_DATA_DIRS`` has been updated
+to match the freedesktop spec.
```

Comment:
I think an API change is when a new parameter is added, or a new error is raised - something that the user should be conscious of when thinking about the inputs and outputs of the code they're writing. I don't think this change qualifies.

I'm not actually not convinced it needs to be in the release notes at all - but if you think it should be, I'd suggest putting it in 'Other Changes'

Reply:
Looking back at older release notes, yeah, it doesn't seem to fit inside the "API changes" section, which matches your definition. It could fit under "Other changes", but since the Linux font finding was undocumented behavior, I think its fine to not include it. I'll remove it later when I make the rest of changes.